bitrise/bitrise.go

```diff
@@ -425,7 +426,7 @@ func (app App) WaitForBuilds(buildSlugs []string, statusChangeCallback func(buil
 				continue
 			}
 
-			if build.Status != 1 {
+			if build.Status != 1 && build.Status != 4 {
```

Comment:
Could you please add a comment here explaining what each number represents (and the one on line 424 as well)?

Reply:
Good call, I was going to link some documentation, but haven't found any public one. I raised this with tech writers, but in the meantime I refactored it a bit to make it easier to understand what's going on. Wdyt?